server/engine/engine.py

```diff
@@ -612,7 +634,7 @@ def processLocalSubjectFolder(path: str, outputName: str = None, href: str = '')
     nimble.biomechanics.OpenSimParser.saveOsimScalingXMLFile(
         'optimized_scale_and_markers', finalSkeleton, massKg, heightM, 'Models/unscaled_but_with_optimized_markers.osim', 'Unassigned', 'Models/optimized_scale_and_markers.osim', path + 'results/Models/rescaling_setup.xml')
     # 8.2.3. Call the OpenSim scaling tool
-    command = 'cd '+path+'results && opensim-cmd run-tool ' + \
+    command = 'cd '+path+'results && opensim-cmd.exe run-tool ' + \
```

Comment:
This is interesting. I suppose it's true that most users are on Windows. Are they going to have `opensim-cmd.exe` on their path? Perhaps we should add instructions to the README about that.

Reply:
Oh crap, this was not supposed to make it in here, this is just a quirk with my local environment on WSL. Removing!